packages/vue/src/components/primitives/amplify-button.vue

```diff
@@ -0,0 +1,29 @@
+<script setup lang="ts">
+interface PropsInterface {
+  type?: 'submit' | 'button';
+  fullWidth?: boolean | string;
+  size?: 'small' | 'medium' | 'large';
+  variation?: 'primary' | 'default' | 'link';
+  fontWeight?: 'normal' | 'bold' | 'lighter';
+}
+const { type, fullWidth, size, variation, fontWeight } = withDefaults(
+  defineProps<PropsInterface>(),
+  {}
+);
+</script>
+<template>
+  <slot v-bind="$attrs" name="buttont">
+    <button
+      :type="type"
```

Comment:
Can we add 
```suggestion
    <button
      class="amplify-button"
      :type="type"
```
so these are styled by default?

Reply:
Sure, just pushed